ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -95,6 +100,101 @@ def format_locator(locator, string_to_insert):
     return locator[0].format(string_to_insert), locator[1]
 
 
+def create_storage_class_ui(
```

Comment:
Do we plan on using this function to create SCs with different reclaim policies and provisioners as well? Since the current implementation only creates an SC using the RBD provisioner with reclaim policy delete.

Reply:
updated, please check